docs/nodes/setup/standalone/manual.md

```diff
@@ -30,7 +30,7 @@ git checkout mainnet-0.9.0 && make clean install
 To compile and install, run:
 
 ```console
-git checkout v0.13.0 && make clean install
+git checkout v0.13.2 && make clean install
```

Comment:
It should be the live version. `v0.13.2` isn't live yet, so it should be updated to `v0.13.1`.

Reply:
it's 0.13.3 now :)